docs.bruno/testing.bru

```diff
@@ -0,0 +1,11 @@
+meta {
```

Comment:
Consider adding comments to explain what each section of the testing file does. This can make it easier for other developers to understand the purpose of the test. [medium]